vms/components/fee/dimensions_test.go

```diff
@@ -426,3 +426,46 @@ func Test_Dimensions_ToGas(t *testing.T) {
 		})
 	}
 }
+
+func Benchmark_Dimensions_Add(b *testing.B) {
+	require := require.New(b)
+
+	lhs := Dimensions{600, 10, 10, 1000}
+	rhs := []Dimensions{
+		{1, 1, 1, 1},
+		{10, 10, 10, 10},
+		{100, 100, 100, 100},
+		{200, 200, 200, 200},
+		{500, 500, 500, 500},
+		{1_000, 1_000, 1_000, 1_000},
+		{10_000, 10_000, 10_000, 10_000},
+	}
+
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		_, err := lhs.Add(rhs...)
+		require.NoError(err)
```

Comment:
This does a few things under the hood that may add non-negligible time on top of the benchmarked function. Consider a raw `if err != nil { b.Error...`.

Reply:
Done

Reply:
Based on the above benchmarks diffs, it seems like we are essentially benchmarking `require.NoError` rather than benchmarking `Add`.

imo - we should drop the error handling entirely. We have correctness checking in the tests for these methods already.